pkg/config/v2/config.go

```diff
@@ -93,6 +93,19 @@ func (c *cfg) SetProjectName(string) error {
 	}
 }
 
+// GetCLIVersion implements config.Config
+func (c cfg) GetCLIVersion() string {
```

Comment:
We should not change the config/v2 it is deprecated and the changes should be addressed into the config/v3